package.json

```diff
@@ -14,51 +14,69 @@
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "ember test",
-    "test:all": "ember try:each"
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.10.0",
-    "ember-modifier-manager-polyfill": "^1.1.0"
+    "ember-cli-babel": "^7.26.6",
+    "ember-modifier-manager-polyfill": "^1.2.0"
   },
   "devDependencies": {
-    "@ember/optional-features": "^0.7.0",
-    "ember-cli": "~3.7.1",
-    "ember-cli-dependency-checker": "^3.0.0",
-    "ember-cli-htmlbars": "^3.0.0",
-    "ember-cli-htmlbars-inline-precompile": "^2.1.0",
-    "ember-cli-inject-live-reload": "^2.0.1",
+    "@ember/optional-features": "^2.0.0",
+    "@ember/test-helpers": "^2.4.0",
+    "@embroider/test-setup": "^0.43.5",
+    "@glimmer/component": "^1.0.4",
+    "@glimmer/tracking": "^1.0.4",
+    "babel-eslint": "^10.1.0",
+    "ember-auto-import": "^1.11.3",
+    "ember-cli": "~3.27.0",
+    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-htmlbars": "^5.7.1",
+    "ember-cli-inject-live-reload": "^2.1.0",
+    "ember-cli-terser": "^4.0.2",
     "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-load-initializers": "^2.0.0",
+    "ember-load-initializers": "^2.1.2",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-qunit": "^4.2.0",
-    "ember-resolver": "^5.0.1",
-    "ember-source": "~3.7.0",
-    "ember-source-channel-url": "^1.1.0",
-    "ember-template-lint": "^1.1.0",
-    "ember-try": "^1.0.0",
-    "eslint": "^5.12.1",
-    "eslint-config-prettier": "^4.0.0",
-    "eslint-plugin-ember": "^6.1.0",
-    "eslint-plugin-node": "^8.0.1",
-    "eslint-plugin-prettier": "^3.0.1",
+    "ember-page-title": "^6.2.2",
+    "ember-qunit": "^5.1.4",
+    "ember-resolver": "^8.0.2",
+    "ember-source": "~3.28.0",
+    "ember-source-channel-url": "^3.0.0",
+    "ember-template-lint": "^3.6.0",
+    "ember-try": "^1.4.0",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-ember": "^10.5.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.4.0",
+    "eslint-plugin-qunit": "^6.2.0",
     "loader.js": "^4.7.0",
-    "prettier": "^1.16.1",
-    "qunit-dom": "^0.8.0",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^2.3.2",
+    "qunit": "^2.16.0",
+    "qunit-dom": "^1.6.0",
     "release-it": "^12.2.1",
     "release-it-lerna-changelog": "^1.0.3"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "10.* || >= 12"
```

Comment:
👍 .  We might want to bump this all the way to 12 so we don't have to do another major after this one.  HOwever, this might mean we need the test suite to run with min 3.10.

https://github.com/ember-cli/ember-cli/blob/master/docs/node-support.md